core/src/v2/schema.rs

```diff
@@ -190,6 +190,12 @@ impl_type_simple!(
     DataType::File,
     DataTypeFormat::Binary
 );
+#[cfg(feature = "actix-session")]
+impl_type_simple!(
+    actix_session::Session,
+    DataType::File,
+    DataTypeFormat::Binary
+);
```

Comment:
Let's do an empty impl instead? Because it doesn't feel right to have this schema for this type.

Reply:
What do you think to extend the macro to also support the following syntax?
I know it's doing nothing, but at least it's consistent!

```
impl_type_simple!(actix_session::Session);
```

Reply:
I just did it... if you are unhappy with this, then I'll revert and do the impl separately.

Reply:
No, that's okay. It's good to have it. :smile: